src/main/java/com/epam/tc/hw10/services/RestTrelloService.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.tc.hw10.services;
+
+import com.epam.tc.hw10.controllers.BoardRequests;
+import com.epam.tc.hw10.controllers.CardRequests;
+import com.epam.tc.hw10.controllers.ListRequests;
+import com.epam.tc.hw10.dto.BoardDto;
+import com.epam.tc.hw10.dto.CardDto;
+import com.epam.tc.hw10.dto.ListDto;
+import io.restassured.response.Response;
+
+public class RestTrelloService {
```

Comment:
why u create this call. u already have BoardRequests, CardRequests classes... 

one class with methods for all controllers - is too much and will be unsupportable in the future